src/wirecloud/defaulttheme/static/css/workspace/widget.scss

```diff
@@ -93,7 +93,6 @@
 }
 
 .wc-widget-footer {
-    height: 3px;
```

Comment:
From what I can see this is not necessary and only creates a white border at the bottom